src/pivotal-ui/components/pui-variables.scss

```diff
@@ -461,8 +461,16 @@ $table-scrollable-body-border:   1px solid $gray-8;
 // Buttons
 // -------------------------
 
-$btn-font-weight:                600 !default;
+$btn-font-weight:                400 !default;
```

Comment:
Cool. Just the kind of thing that'd be great to call out, not just slip in in what looks like an isolated change.